libs/react-helpers/src/lib/generic-data-provider.spec.ts

```diff
@@ -5,6 +5,7 @@ import {
 } from './generic-data-provider';
 import type {
   CombineDerivedData,
+  CombineDerivedDelta,
```

Comment:
thought (non-blocking): MIght be nice to have a test in here for the new behaviour introduced in generic data provider

Reply:
Tests are on first place on post merge tasks list.